<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorizationForm</class>
 <widget class="QDialog" name="AuthorizationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuthorizationForm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="loginLabel">
       <property name="text">
        <string>Логин:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="loginLineEdit"/>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="showHideBtn">
       <property name="text">
        <string>Показать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="enterBtn">
       <property name="text">
        <string>Войти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="registrationBtn">
       <property name="text">
        <string>Регистрация</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>loginLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>enterBtn</tabstop>
  <tabstop>showHideBtn</tabstop>
  <tabstop>registrationBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>registrationBtn</sender>
   <signal>clicked()</signal>
   <receiver>AuthorizationForm</receiver>
   <slot>createAccountSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enterBtn</sender>
   <signal>clicked()</signal>
   <receiver>AuthorizationForm</receiver>
   <slot>checkLoginPassSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showHideBtn</sender>
   <signal>clicked()</signal>
   <receiver>AuthorizationForm</receiver>
   <slot>showHidePasswordSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createAccountSlot()</slot>
  <slot>checkLoginPassSlot()</slot>
  <slot>showHidePasswordSlot()</slot>
 </slots>
</ui>
